### DISCUSSION TITLE
Allow to disable autoformat text to emoticons :) -> 😀

### DISCUSSION THREAD

**Discussion by swarzynski** [zim-desktop-wiki/zim-desktop-wiki#2471] [Ideas]
Hello!

I haven't found any possibility to disable auto changing text to emoticons, e.g. :) -> 😀

Maybe there is or should be option to disable it?
That would be useful. Now it is annoying for me :)

**Comment by introt** [comment]
The substitution is done by the Insert Symbol plugin. You can either disable it, or remove that substitution from the list:

1. Insert > Symbol...
2. Edit
3. Add a `#` to the beginning of the line containing `:)` 
4. Save the file
5. Restart Zim

https://github.com/zim-desktop-wiki/zim-desktop-wiki/blob/c88cf3cb53896bf272e87704826b77e82eddb3ef/data/symbols.list#L349

Have a nice day ;)
